mim/commands/gridsearch.py

```diff
@@ -346,6 +346,9 @@ def gridsearch(
             ] + common_args
         elif launcher == 'slurm':
             parsed_srun_args = srun_args.split() if srun_args else []
+            if not any(['--job-name' in x for x in parsed_srun_args]):
+                job_name = osp.splitext(config_path.split('/')[-1])[0]
```

Comment:
should we use `os.path.split()` to split the config_path to support the windows environment?